source/_data/SymbioticLab.bib

```diff
@@ -1205,3 +1205,39 @@ @Article{orloj:arxiv22
   state-of-the-art.
   }
 }
+
+@InProceedings{modelkeeper:nsdi23,
+  author    = {Fan Lai and Yinwei Dai and Harsha V. Madhyastha and Mosharaf Chowdhury},
+  booktitle = {USENIX NSDI},
+  title     = {ModelKeeper: Accelerating DNN Training via Automated Training Warmup},
```

Comment:
Should be`{ModelKeeper}` and `{DNN}` (i.e. add brackets) to preserve capitalization. I only learned about this recently, and we probably should keep this convention for all publications at least from now on.

Reply:
Thanks!